### PR TITLE
add directions page and add directions link to Hero

### PR DESCRIPTION
**Pull request by nondescryptid**

### CODE REVIEW

File: src/pages/directions.astro [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+<Layout title="Directions">
+    <section>
+      <h1>Directions for Geekcamp 2024</h1>
+      Geekcamp 2024 is at <b>SMU Connexion, Level 5</b>. 
+      <h3>
+        1. City Hall (Exit A) - step-free access
+      </h3>
+      <p>
+        Get up to street level via Exit A. You can take the escalator or a lift. Exit Raffles City by the door on the left side. 
+        <img src="/src/data/images/directions2024/cityhall/raffles-city.jpg"/>
```
Comment (serverwentdown) [inline src/pages/directions.astro]:
```suggestion
        <Image src="/data/images/directions2024/cityhall/raffles-city.jpg"/>
```

I think it should be this

Comment (serverwentdown) [reply src/pages/directions.astro]: https://docs.astro.build/en/guides/images/

File: src/pages/directions.astro [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        You should see Coffee Bean and Tea Leaf  and a large white gift box in the distance. 
+        <img src="/src/data/images/directions2024/cityhall/gift-box.jpg"/>
```
Comment (serverwentdown) [inline src/pages/directions.astro]:
```suggestion
        <Image src="/data/images/directions2024/cityhall/gift-box.jpg"/>
```

**Review by serverwentdown — CHANGES_REQUESTED** [review]

File: src/pages/directions.astro (hunk 1 above)
Comment (nondescryptid) [reply src/pages/directions.astro]: thanks Ambrose 😄  I've updated the images to use the astro Image component in line with the docs you sent and also this: https://docs.astro.build/en/reference/modules/astro-assets/#image-properties (for the Image component, alt-text is also required)

**Review by saumilthecode — APPROVED** [review]
works in preview 👍 

lgtm

**Comment by saumilthecode** [comment]
we're 2 days away from geekcamp so im merging in now

**Comment by saumilthecode** [comment]
and we await ambrose's review (i dare not fight the git)

**Review by serverwentdown — APPROVED** [review]

**Pull request merged**